packages/ui/src/Tooltip/Tooltip.tsx

```diff
@@ -33,7 +33,7 @@ export function Tooltip({ children, description, side = 'top' }: TooltipProps) {
             sideOffset={GAP_BETWEEN_TOOLTIP_AND_TRIGGER}
             side={side}
             role="tooltip"
-            className="tooltip-content tooltip-animation"
+            className="tooltip-base tooltip-content tooltip-animation"
```

Comment:
This is a nit - but I feel like it would be nice to extract this into a constant and then import it in `Popover` - it would ensure that they're always in sync.

Reply:
Here's a [a1f558f](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/1181/commits/a1f558f87147c283b2b18466d55e719e86f09e92) - This actually helps us not have classes in two different files (except for the animation, which we could still move to a shared globals file if we wanted as some point.)